index.js

```diff
@@ -58,6 +61,55 @@ const TransportStream = module.exports = function TransportStream(options = {})
  */
 util.inherits(TransportStream, Writable);
 
+/**
+ * Returns true if the info log level means it should be logged
+ * @param {mixed} info - TODO: add param description.
+ * @returns {boolean} - TODO: add returns description.
```

Comment:
PR leaves some TODOs open here.

Reply:
These TODOs are inherited from the original method that I extracted the code from. I just split the method out to fix the worst eslint issues that were breaking the ```npm run test```. I didn't want to assume what the descriptions are as I'm not super familiar with the code and project nomenclature.

Reply:
I had looked in the diff for that but didn't see a corresponding line being moved from elsewhere.  Where was it extracted from?
In any case, those TODOs are added automatically by some JSDoc support tools in IDE packages.  With the comment in the line above, these two lines can probably just be removed.